docs/source/kinds_of_types.rst

```diff
@@ -514,7 +544,7 @@ string-literal types with non-string-literal types freely:
 
    class A: pass
 
-String literal types are never needed in ``# type:`` comments.
+String literal types are never needed in `# type:` comments.
```

Comment:
Or in stub files (not directly related to the topic of this PR, but we might as well add it).

Reply:
Mentioned stub files